Form-Controls/index.html

```diff
@@ -1,28 +1,97 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <title>My form exercise</title>
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
-
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
-
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+
+
+                           <!-- FORM SECTION -->
+
+
+    <header>
+      <h1>Active Mens T-shirt</h1>
+    </header>
+
+    <p>ORDER NOW</p>
+
+    <main>
+      <form>
+         <div>
+          <label for="first-name"><span class="required"></span> First Name</label>
+          <input type="text" id="first-name" name="first-name" placeholder="First Name" required minlength="2"/>
+         </div>
+
+         <div>
+          <label for="last-name"><span class="required"></span> Last Name</label>
+          <input type="text" id="last-name" name="last-name" placeholder="Last Name" required minlength="2"/>
+         </div>
+
+        <div>
+          <label for="email"><span class="required"></span> Email Address</label>
+          <input type="email" name="email" id="email" placeholder="Email"/>
+        </div>
+
+        <div>
+          <label for="color">Color</label>
+        
+          <select name="color" id="color">
+            <option value="cream">Red</option>
+            <option value="black">Black</option>
+            <option value="blue">Blue</option>
+            <option value="cream">White</option>
+            <option value="black">Yellow</option>
+            <option value="blue">Brown</option>
+          </select>
+        </div>
+
+        <div>
+          <label for="size">Size</label>
+
+          <select for="size" id="size">
```

Comment:
As above, the requirement was " I require one size from a defined set of 6 sizes." I only count 5.